.github/workflows/cross_platform_ci.yml

```diff
@@ -0,0 +1,105 @@
+name: MoveIt core cross-platform RoboStack build
```

Comment:
I'm not sure we want to sync this, @henningkayser what do I do if I want to drop this commit?  Should I do an interactive rebase and delete the commit this came from?